models/contacts.js

```diff
@@ -1,19 +1,85 @@
-// const fs = require('fs/promises')
+const fs = require("fs/promises");
+const path = require("path");
+const { nanoid } = require("nanoid");
 
-const listContacts = async () => {}
+const contactsPath = path.resolve("./models/contacts.json");
 
-const getContactById = async (contactId) => {}
+// GET - Get all contacts
 
-const removeContact = async (contactId) => {}
+const listContacts = async () => {
+  try {
+    const data = await fs.readFile(contactsPath, "utf-8");
+    return JSON.parse(data);
+  } catch (error) {
+    console.error(`Error: ${error.message}`);
+  }
+};
 
-const addContact = async (body) => {}
+// GET - Find contact by id
 
-const updateContact = async (contactId, body) => {}
+const getContactById = async (contactId) => {
+  const contactsArr = await listContacts();
+  const contact = contactsArr.find(
+    (element) => element.contactId === contactId
+  );
```

Comment:
Dosc dlugie. Mozesz krocej:
```js
const getContactById = async (id) => {
  const contacts = await listContacts();
  const contact = contacts.find(({contactId}) => contactId === id);
```

Reply:
nie ma tez potrzeby mowic `contactsArr`, bo liczba mnoga od razu mowi nam, ze to jest array.

Reply:
Racja, poprawiłem